mvc/src/main/java/webmvc/org/springframework/web/servlet/view/JsonView.java

```diff
@@ -1,14 +1,47 @@
 package webmvc.org.springframework.web.servlet.view;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import web.org.springframework.http.MediaType;
 import webmvc.org.springframework.web.servlet.View;
 
+import java.io.PrintWriter;
 import java.util.Map;
+import java.util.NoSuchElementException;
 
 public class JsonView implements View {
 
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
     @Override
-    public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+    public void render(final Map<String, ?> model,
+                       final HttpServletRequest request,
+                       final HttpServletResponse response) throws Exception {
+        final String json = getJsonString(model);
+        response.setContentType(MediaType.APPLICATION_JSON_UTF8_VALUE);
+        final PrintWriter writer = response.getWriter();
+        writer.write(json);
+        writer.flush();
+    }
+
+    private String getJsonString(Map<String, ?> model) {
```

Comment:
```suggestion
    private String getJsonString(final Map<String, ?> model) {
```
사소하지만 컨벤션으로 가져가시는 것 같아서 언급해봅니다~

Reply:
기존 코드에 컨벤션을 추가로 지키기가 쉽지않군요 껄껄.. 자세한 리뷰 감사합니다 조이!